File: @LV1/1.8/dasblinkenlights/dasblinkenlights.go
package main

import "fmt"

func gcd(a int, b int) int {
	if b == 0 {
		return a
	}
	return gcd(b, a%b)
}

func main() {
	var a, b, tl int
	fmt.Scanf("%d %d %d", &a, &b, &tl)

	lcm := a * b / gcd(a, b)

	if tl/lcm >= 1 {
		fmt.Println("yes")
	} else {
		fmt.Println("no")
	}
}

File: @LV1/1.8/dasblinkenlights/dasblinkenlights.cpp
#include <cstdio>

int gcd(int a, int b) {
    if (b == 0) {
        return a;
    }
    return gcd(b, a % b);
}

int main() {
    int a, b, tl;
    scanf("%d %d %d", &a, &b, &tl);

    int lcm = a * b / gcd(a, b);

    if (tl / lcm >= 1) {
        puts("yes");
    } else {
        puts("no");
    }
}